frontend/revive-gallery-app/src/components/home/Navbar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import logoImage from '../../static/images/revive-logo.png';
+import { useUser } from '../auth/UserContext';
+import '../../styles/home/Navbar.css'; 
+
+const Navbar = () => {
+    const { user, token } = useUser();
+  
+    return (
+      <nav className="navbar">
+        <div className="navbar-logo">
+          <Link to="/">
+            <img src={logoImage} alt="Revive Gallery Logo" />
+          </Link>
+        </div>
+        <ul className="navbar-links">
+          <li>
+            <Link to="/">Home</Link>
+          </li>
+          <li>
+            <Link to="/products">Products</Link>
+          </li>
+          <li>
+            <Link to="/products">Categories</Link>
+          </li>
+          {user && token ? (
+            <li>
+              <Link to="/addproduct">Add Product</Link>
```

Comment:
@anjaliputta It would be better to have Add Product button in user profile dropdown .

Reply:
@SaiKiran0407 I think we will have lot of options in user profile dropdown such as my products, my orders, my score, wishlist, cart, logout etc. So I think its more relevant and handy if we place "Add product" outside User profile button on nav bar.

Reply:
@anjaliputta Ok, Sounds good, I am approving.